entrypoint.sh

```diff
@@ -40,3 +40,33 @@ echo "Uploading wheels to anaconda.org..."
 
 anaconda -t $ANACONDA_TOKEN upload --force -u "$ANACONDA_ORG" "$INPUT_ARTIFACTS_PATH"/*.whl
 echo "Index: https://pypi.anaconda.org/$ANACONDA_ORG/simple"
+
+# Get the package name from the "${INPUT_ARTIFACTS_PATH}"/*.whl files
+if [ -d "${INPUT_ARTIFACTS_PATH}" ]; then
+    # Leave off *.whl in ls target to avoid directory name prefix on wheel_name
+    wheel_name=$(ls --almost-all "${INPUT_ARTIFACTS_PATH}" | head --lines 1)
+    IFS=- read package_name discard <<< "${wheel_name}"
+    unset wheel_name
+else
+    echo "No ${INPUT_ARTIFACTS_PATH} directory found."
+    exit 1
+fi
+
+if [ ! -z "${INPUT_N_LATEST_UPLOADS}" ]; then
+  # Remove all _but_ the last "${INPUT_N_LATEST_UPLOADS}" package versions
+  # N.B.: `anaconda show` places the newest packages at the bottom of
+  # the output of the 'Versions' section and package versions are
+  # preceded with a '   + '.
+  anaconda show "${ANACONDA_ORG}/${package_name}" &> >(grep '+') | \
+      sed 's/.* + //' | \
+      head --lines "-${INPUT_N_LATEST_UPLOADS}" > remove-package-versions.txt
```

Comment:
c.f. https://github.com/matplotlib/matplotlib/pull/23349 for a detailed breakdown of what is happening here.